### DISCUSSION TITLE
Project collections

### DISCUSSION THREAD

**Discussion by triphora** [modrinth/.github#30] [Feature requests]
This is a tracking issue sort of thing for whenever we get around to this. Project collections are a planned feature that will allow users to create a collection of related projects. Think YouTube playlists, but for Modrinth projects. modrinth/labrinth#277

There would likely be built-in ones for stuff like a Watchlist, a shopping cart-esque feature (see modrinth/knossos#559), one for followed projects, and so on.

**Comment by NEMOxd2010** [comment]
it should probably look something like that
![Screenshot 2023-06-08 170919](https://github.com/modrinth/.github/assets/115355838/4241bf10-59e7-4ab2-9652-b6534bedb71c)

**Comment by Clozent** [comment]
It would be cool if collections had an option to automatically add missing dependencies to it.

**Comment by triphora** [comment]
Closing since this has been implemented.
